src/raster/r.mess/r.mess.py

```diff
@@ -192,9 +193,19 @@ def main(options, flags):
         return 0
 
     # Reference / sample area or points
-    ref_rast = options["ref_rast"]
     ref_vect = options["ref_vect"]
-    if ref_rast:
+    if bool(ref_vect):
+        topology_check = gs.parse_command("v.info", map=ref_vect, flags="t")
```

Comment:
There is [`gs.vector_info_topo`](https://grass.osgeo.org/grass83/manuals/libpython/script.html#script.vector.vector_info_topo) function you could use here.

Reply:
Thanks, done.